app/assets/javascripts/discourse/app/components/topic-list/latest-topic-list-item.gjs

```diff
@@ -0,0 +1,110 @@
+import Component from "@glimmer/component";
+import { concat, hash } from "@ember/helper";
+import { service } from "@ember/service";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import PostsCountColumn from "discourse/components/topic-list/posts-count-column";
+import {
+  navigateToTopic,
+  showEntrance,
+} from "discourse/components/topic-list-item";
+import TopicPostBadges from "discourse/components/topic-post-badges";
+import TopicStatus from "discourse/components/topic-status";
+import UserAvatarFlair from "discourse/components/user-avatar-flair";
+import UserLink from "discourse/components/user-link";
+import avatar from "discourse/helpers/avatar";
+import categoryLink from "discourse/helpers/category-link";
+import concatClass from "discourse/helpers/concat-class";
+import discourseTags from "discourse/helpers/discourse-tags";
+import formatDate from "discourse/helpers/format-date";
+import topicFeaturedLink from "discourse/helpers/topic-featured-link";
+import topicLink from "discourse/helpers/topic-link";
+
+export default class LatestTopicListItem extends Component {
+  @service appEvents;
+
+  showEntrance = showEntrance;
+  navigateToTopic = navigateToTopic;
+
+  click(e) {
+    // for events undefined has a different meaning than false
+    if (this.showEntrance(e) === false) {
+      return false;
+    }
+
+    return this.unhandledRowClick(e, this.args.topic);
```

Comment:
I don't see this function defined in this class?

Reply:
Good catch, had to copy the temporary implementation from topic-list-item.